internal/proxy/add.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# add.sh adds new dependencies to the Go Cloud module proxy. This script
+# should be run from the root of the repository.
+
+set -euo pipefail
+
+function log() {
+  echo "add.sh: $@" 1>&2
+}
+
+function die() {
+  log "$@"
+  exit 1
+}
+
+rsyncing=1
```

Comment:
You could set `rsyncing` to `true`/`false` and then just write `if $rsyncing; ...`.

But if you prefer this way, that's fine with me.

Reply:
Done.